src/main/java/org/cascadebot/cascadebot/utils/votes/VoteButtonGroupBuilder.java

```diff
@@ -57,12 +58,12 @@ public VoteButtonGroupBuilder(VoteMessageType type) {
      * @return this.
      */
     public VoteButtonGroupBuilder setOptionsAmount(int amount) {
-        if (type == VoteMessageType.YES_NO || type == VoteMessageType.CUSTOM) {
-            throw new UnsupportedOperationException("Cannot set options amount for yes no votes, or custom votes");
+        if (type == VoteMessageType.YES_NO) {
+            throw new UnsupportedOperationException("Cannot set options amount for yes no votes");
         }
 
-        if (type == VoteMessageType.LETTERS && amount > 26) {
-            throw new UnsupportedOperationException("Cannot have more then 26 options when using letters");
+        if (type == VoteMessageType.LETTERS && amount > 15) {
```

Comment:
Could you just leave a note about why this is 15? Just for future reference (We will forget)